flink-table/flink-table-runtime-blink/src/test/java/org/apache/flink/table/runtime/sort/BinaryExternalSorterTest.java

```diff
@@ -76,11 +75,14 @@ public BinaryExternalSorterTest(
 
 	@Parameterized.Parameters(name = "spillCompress-{0} asyncMerge-{1}")
 	public static Collection<Boolean[]> parameters() {
-		return Arrays.asList(
-				new Boolean[]{false, false},
-				new Boolean[]{false, true},
-				new Boolean[]{true, false},
-				new Boolean[]{true, true});
+		// To avoid too long testing time, we only tested the default configuration.
+		// If BinaryExternalSorter has changed, please open it manually for testing.
+//		return Arrays.asList(
+//				new Boolean[]{false, false},
+//				new Boolean[]{false, true},
+//				new Boolean[]{true, false},
+//				new Boolean[]{true, true});
```

Comment:
Manually testing is hard to guarantee the correctness. I think we need to find a way to add this to nightly tests in this pull request.